app/lectures/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+from rest_framework import routers
+
+from .views import LectureViewList, LectureView
+
+"""
+Django REST url
+"""
+
+router = routers.DefaultRouter()
+router.register("", LectureViewList, basename="lectures")
+urlpatterns = [path("new/", LectureView.as_view())]
```

Comment:
Why does lecture has "new" path? I guess we need reference a specific lecture by a id.